.gitignore

```diff
@@ -53,5 +54,5 @@ docs/modules.rst
 node_modules
 
 # webpack built files
-*stats.json
+kolibri/**/build/*
 kolibri/**/static/*
```

Comment:
I understand why this is necessary here, but the wildcard might cause brief confusion if someone decides to call a directory 'static' eventually